core/src/OpenACC/Kokkos_OpenACCSpace.hpp

```diff
@@ -85,16 +85,26 @@ class OpenACCSpace {
 template <>
 struct Kokkos::Impl::MemorySpaceAccess<Kokkos::HostSpace,
                                        Kokkos::Experimental::OpenACCSpace> {
+#if defined(KOKKOS_ENABLE_OPENACC_FORCE_HOST_AS_DEVICE)
+  enum : bool{assignable = true};
+  enum : bool{accessible = true};
+#else
```

Comment:
Do you really want to allow accessing `OpenACCSpace` from the host? This will likely break code when you are first forcing OpenACC to target the host and later a proper device.

Reply:
This is based on an assumption that if a user explicitly specify `KOKKOS_ENABLE_OPENACC_FORCE_HOST_AS_DEVICE` but without a NVIDIA/AMD GPU architecture (`KOKKOS_ARCH`), all the Kokkos parallel constructs will be executed on the host CPU. If the user wants to use both CPU and GPU as offloading targets, the user should not specify `KOKKOS_ENABLE_OPENACC_FORCE_HOST_AS_DEVICE`.